.changeset/shy-elephants-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Add OptionalIntlProvider to KaizenProvider
```

Comment:
Please include some description of the feature which adding this supports. ([more info](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3184164865/Chapter+4+The+Release#Writing-a-changeset))

Reply:
Okay, I've added some more detail